Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM centos:7.5.1804
+
+COPY . /etc/hammers
```

Comment:
I would move this to after the yum package installs. Reason being, any change to hammers will invalidate all the layers containing package installs, which need not happen (packages shouldn't be changing really, just the hammers code and pip dependencies.)

So, suggestion is to do something like

```Dockerfile
RUN yum install ...

COPY . /etc/hammers
RUN pip install -r /etc/hammers/requirements.txt
```